rotkehlchen/chain/evm/accounting/aggregator.py

```diff
@@ -155,6 +164,40 @@ def get_accounting_settings(self, pot: 'AccountingPot') -> dict[str, TxEventSett
             take=1,
             method='acquisition',
         )
+        swap_key = str(HistoryEventType.TRADE) + '__' + str(HistoryEventSubType.SPEND)
+        result[swap_key] = TxEventSettings(
+            taxable=True,
+            count_entire_amount_spend=False,
+            count_cost_basis_pnl=True,
+            method='spend',
+            take=2,
+            multitake_treatment=TxMultitakeTreatment.SWAP,
+        )
+        airdrop_key = str(HistoryEventType.RECEIVE) + '__' + str(HistoryEventSubType.AIRDROP)
+        airdrops_taxable = LedgerActionType.AIRDROP in pot.settings.taxable_ledger_actions
+        result[airdrop_key] = TxEventSettings(
+            taxable=airdrops_taxable,
+            count_entire_amount_spend=False,
+            count_cost_basis_pnl=False,
+            method='acquisition',
+            take=1,
+        )
+        reward_key = str(HistoryEventType.RECEIVE) + '__' + str(HistoryEventSubType.REWARD)
```

Comment:
We already have this in aave1-v1, compound and votium. The only difference is aave-v1 has `count_cost_basis_pnl` as True.

The question here is why the discrepancy? Am I correct in seeing that **for acquisitions**, `count_cost_basis_pnl` is not used? So it's the same. So then again the question on why have them both in the accountants and here stays.